test/unittests/commands/validate_data/test_make_data_rdf.py

```diff
@@ -264,5 +269,31 @@ def test_none(self, uri_value_deserialised_none: UriValueDeserialised) -> None:
         assert val.object_value == Literal("", datatype=XSD.string)
 
 
+@pytest.mark.parametrize(
+    ("value", "expected"),
+    [
+        (BitstreamDeserialised("id", "test.jpg"), "jpg"),
+        (BitstreamDeserialised("id", None), "No file path was given."),
+        (IIIFUriDeserialised("id", "https://uri.org"), "https://uri.org"),
+        (IIIFUriDeserialised("id", None), "None"),
+    ],
+)
+def test_transform_file_value(value: AbstractFileValueDeserialised, expected: str) -> None:
+    assert _transform_file_value(value).value == Literal(expected)
+
+
+@pytest.mark.parametrize(
+    ("value", "expected"),
+    [
+        ("test.jpg", "jpg"),
+        ("test.JPG", "jpg"),
+        (None, "No file path was given."),
+        ("test", "This file is missing a valid extension, actual value: test"),
```

Comment:
This ensures that even if a valid extension without a file is entered it will fail the validation.